corehq/toggles.py

```diff
@@ -441,7 +440,7 @@ def toggles_dict(username=None, domain=None):
     'Enable tracking ownership cleanliness on submission',
     TAG_EXPERIMENTAL,
     randomness=.05,
-    namespace=NAMESPACE_DOMAIN,
+    namespaces=[NAMESPACE_DOMAIN],
```

Comment:
this is also a fix:

![image](https://cloud.githubusercontent.com/assets/66555/7609916/7669add8-f978-11e4-9c19-4cc0b3932f50.png)